addons/mail/static/src/js/form_view.js

```diff
@@ -1,26 +0,0 @@
-odoo.define('mail.FormView', function (require) {
-"use strict";
-
-var FormView = require('web.FormView');
-
-/**
- * This file is used to add "message_attachment_count" to fieldsInfo so we can fetch its value for the
- * chatter's attachment button without having it explicitly declared in the form view template.
```

Comment:
I think this potentially avoided one RPC? If so we need to somehow find a solution at the end.

Reply:
Not really. In fact we just do a rpc call to get all attachment ids directly instead of waiting for the user to click on attachment button. Doing so, we can get the amount of attachment that way so this field is not really required anymore.